src/config/oauth.json

```diff
@@ -4,6 +4,10 @@
   "api_server_base_path": "{{ .Env.K8S_API_SERVER_BASE_PATH }}",
   "ws_api_server": "{{ .Env.WS_K8S_API_SERVER }}",
   "openshift_console_url": "{{ .Env.OPENSHIFT_CONSOLE_URL }}",
+  "forge_api_url": "{{ .Env.FABRIC8_FORGE_API_URL }}",
```

Comment:
oauth.json is for fabric8-runtime-console, for fabric8-ui please use https://github.com/jstrachan/fabric8-ui/blob/999d4a6b13bc2ee3e059ae78d1b2571dee8b8781/src/config/fabric8-ui.config.json